protocol/v2/ssv/validator/committee.go

```diff
@@ -290,6 +315,27 @@ func (c *Committee) ProcessMessage(logger *zap.Logger, msg *queue.SSVMessage) er
 	return nil
 
 }
+func (c *Committee) unsafePruneExpiredRunners(logger *zap.Logger, currentSlot phase0.Slot) error {
+	if runnerExpirySlots > currentSlot {
+		return nil
+	}
```

Comment:
could be we never reach this piece of code?
since runnerExpirySlots is hardcoded to `phase.Slot(34)`